timesketch/frontend-ng/src/components/Visualization/ChartConfig.vue

```diff
@@ -0,0 +1,236 @@
+<!--
+Copyright 2024 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-container class="ma-0">
+    <v-row>
+      <v-col>
+        <TsChartTypeSelect
+          :aggregator="aggregatorType"
+          :chart="selectedChartType"
+          @selectedChartType="selectedChartType = $event"
+          required
+          :rules="[rules.required]"
+        >
+        </TsChartTypeSelect>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col>
+        <v-text-field
```

Comment:
What do you think about removing the name field by default and only ask for a name when the user hits the Save button in a small dialog (similar to how we do saved searches)?

Reply:
At the create visualization stage it isn't a mandatory field to be able load data to preview and you cannot save without specifying a title so I'm not sure if breaking it out into a separate dialog makes a big difference.  Having said that, I have no strong opinions and can see if I can create a dialog if that's the direction you would like to take.

Reply:
Solved by the UI mock discussed offline.